src/Nav.tsx

```diff
@@ -28,6 +27,30 @@ import AuthorJpg from "./images/nav/author.jpg";
 import SteamJpg from "./images/nav/steam.jpg";
 
 import SourceIcon from "@mui/icons-material/Source";
+import { OverridableComponent } from "@mui/material/OverridableComponent";
+
+class RouteData {
```

Comment:
A simple interface might do here. Nothing is being done with the object state after it is set.